data/pigui/themes/default.lua

```diff
@@ -191,10 +192,10 @@ theme.icons = {
    display_navtarget = 121,
    alert1 = 122,
    alert2 = 123,
-	 ecm_advanced = 124,
-	 systems_management = 125,
-	 distance = 126,
-	 filter = 127,
+   ecm_advanced = 124,
```

Comment:
Same thing here, this file should all be one tab of indentation. If you could update that in a separate commit, I'd really appreciate it.

Reply:
Most likely this file, together with icons.svg, will go to another PR.

Reply:
These files are no longer here.